inlezen (BRK) stand mislukt met oracle database

Bij een nieuwe staging database mislukt het inlezen van een stand bestand met de volgende fout:

`Fout bij inlezen bestand: SQLException: ORA-01400: cannot insert NULL into ("STAGINGTEST"."LAADPROCES"."ID")`

logfile:
```
Fout bij inlezen bestand
nl.b3p.brmo.loader.util.BrmoException: Fout bij loaden basisregistratie gegevens
	at nl.b3p.brmo.loader.BrmoFramework.loadFromStream(BrmoFramework.java:259)
	at nl.b3p.brmo.service.stripes.BasisregistratieFileUploadActionBean.upload(BasisregistratieFileUploadActionBean.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at net.sourceforge.stripes.controller.DispatcherHelper$6.intercept(DispatcherHelper.java:456)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:158)
	at net.sourceforge.stripes.controller.BeforeAfterMethodInterceptor.intercept(BeforeAfterMethodInterceptor.java:113)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:155)
	at org.stripesstuff.plugin.waitpage.WaitPageInterceptor.intercept(WaitPageInterceptor.java:175)
	at net.sourceforge.stripes.controller.ExecutionContext.proceed(ExecutionContext.java:155)
	at net.sourceforge.stripes.controller.ExecutionContext.wrap(ExecutionContext.java:74)
	at net.sourceforge.stripes.controller.DispatcherHelper.invokeEventHandler(DispatcherHelper.java:454)
	at net.sourceforge.stripes.controller.DispatcherServlet.invokeEventHandler(DispatcherServlet.java:278)
	at net.sourceforge.stripes.controller.DispatcherServlet.service(DispatcherServlet.java:160)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:725)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:291)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at net.sourceforge.stripes.controller.StripesFilter.doFilter(StripesFilter.java:260)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:239)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:219)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:106)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:613)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:142)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:79)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:610)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:88)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:537)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1085)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:658)
	at org.apache.coyote.http11.Http11NioProtocol$Http11ConnectionHandler.process(Http11NioProtocol.java:222)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1556)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.run(NioEndpoint.java:1513)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.sql.SQLException: ORA-01400: cannot insert NULL into ("STAGINGTEST"."LAADPROCES"."ID")
Query: INSERT INTO laadproces(bestand_naam,bestand_datum, soort, gebied, opmerking, status,status_datum, contact_email) VALUES (?,?,?,?,?,?,?,?) Parameters: [BURKX01-ZWD03-20140731.zip/BURKX01.xml, 2014-07-31 00:00:00.0, brk, Zeewolde, null, STAGING_OK, 2015-10-12 12:37:35.463, null]
	at org.apache.commons.dbutils.AbstractQueryRunner.rethrow(AbstractQueryRunner.java:392)
	at org.apache.commons.dbutils.QueryRunner.update(QueryRunner.java:491)
	at org.apache.commons.dbutils.QueryRunner.update(QueryRunner.java:404)
	at nl.b3p.brmo.loader.StagingProxy.writeLaadProces(StagingProxy.java:614)
	at nl.b3p.brmo.loader.StagingProxy.loadBr(StagingProxy.java:515)
	at nl.b3p.brmo.loader.BrmoFramework.loadFromStream(BrmoFramework.java:257)
```

--- a/brmo-persistence/db/04_create_triggers_oracle.sql
+++ b/brmo-persistence/db/04_create_triggers_oracle.sql
@@ -1,0 +1,21 @@
+-- create triggers om (conditioneel) een id uit de sequence te halen
+-- fix voor issue #86
+CREATE OR REPLACE TRIGGER LAADPROCES_INSERT_TRIGGER
+        BEFORE INSERT ON LAADPROCES
+        FOR EACH ROW
+BEGIN
+    IF :new.ID IS NULL THEN
+                SELECT LAADPROCES_ID_SEQ.nextval INTO :new.ID FROM DUAL;
+    END IF;
+END;
+
+
+
+CREATE OR REPLACE TRIGGER BERICHT_INSERT_TRIGGER
+        BEFORE INSERT ON BERICHT
+        FOR EACH ROW
+BEGIN
+    IF :new.ID IS NULL THEN
+                SELECT BERICHT_ID_SEQ.nextval INTO :new.ID FROM DUAL;
+    END IF;
+END;

